packages/wallets/src/components/WalletAddedSuccess/WalletAddedSuccess.tsx

```diff
@@ -23,21 +23,19 @@ const WalletAddedSuccess: React.FC<TWalletAddedSuccessProps> = ({
     const { localize } = useTranslations();
     const description = localize('Make a deposit into your new Wallet.');
     const title = useMemo(() => localize('Your {{currency}} wallet is ready', { currency }), [currency, localize]);
-    const renderFooter = useCallback(
-        () => (
-            <div className='wallets-add-more__success-footer'>
-                <WalletButtonGroup isFlex isFullWidth>
-                    <WalletButton onClick={onSecondaryButtonClick} variant='outlined'>
-                        <Localize i18n_default_text='Maybe later' />
-                    </WalletButton>
-                    <WalletButton onClick={onPrimaryButtonClick}>
-                        <Localize i18n_default_text='Deposit' />
-                    </WalletButton>
-                </WalletButtonGroup>
-            </div>
-        ),
-        [onPrimaryButtonClick, onSecondaryButtonClick]
+    const renderFooter = (
+        <div className='wallets-add-more__success-footer'>
+            <WalletButtonGroup isFlex isFullWidth>
+                <WalletButton onClick={onSecondaryButtonClick} variant='outlined'>
+                    <Localize i18n_default_text='Maybe later' />
+                </WalletButton>
+                <WalletButton onClick={onPrimaryButtonClick}>
+                    <Localize i18n_default_text='Deposit' />
+                </WalletButton>
+            </WalletButtonGroup>
+        </div>
     );
```

Comment:
Wouldn't it be better to extract this into its own component? something like `WalletAddedSuccessFooter`? 🤔 
```suggestion
    const renderFooter = (
        <div className='wallets-add-more__success-footer'>
            <WalletButtonGroup isFlex isFullWidth>
                <WalletButton onClick={onSecondaryButtonClick} variant='outlined'>
                    <Localize i18n_default_text='Maybe later' />
                </WalletButton>
                <WalletButton onClick={onPrimaryButtonClick}>
                    <Localize i18n_default_text='Deposit' />
                </WalletButton>
            </WalletButtonGroup>
        </div>
    );
```

Reply:
Updated